### ISSUE TITLE
JFXDatePicker colors

### ISSUE THREAD

**Issue by AhmedMouradDev** [jfoenixadmin/JFoenix#500]
This's a follow up for issue #484 

Hello again,

Thanks for the effort for fixing the issue, but i've just noticed that the same problem happens with the (today / selected date / year) numbers when default color is white as shown below:

(Today is the 4th of nov, selected date is 3rd of nov)
![image](https://user-images.githubusercontent.com/29189251/32400081-8d81e336-c104-11e7-8d5d-73119dea63fa.png)

![image](https://user-images.githubusercontent.com/29189251/32400089-9b77da40-c104-11e7-935e-64b8d13a9153.png)

Also the date icon in the new release gets this weird white/silver background color when focused which didn't use to happen with the previous version i was using (1.7.0):

(focused)
![image](https://user-images.githubusercontent.com/29189251/32400142-1d7941fa-c105-11e7-8bfb-0eaf02ac5987.png)

(unfocused)
![image](https://user-images.githubusercontent.com/29189251/32400153-4270e4ea-c105-11e7-85ed-3272f6191fba.png)

Best regards and thanks for the effort,

**Comment by jfoenixadmin** [comment]
Hello,
well changing the default color changes many nodes in date picker content. 
It would be better solution to change the icon color instead of default color, however this can't be done due to some binding. Will remove the icon binding so it can be changed through css.

Regarding the silver background issue, it's already fixed however was not included in 1.10.0.
you can download the latest version of JFoenix from the website.

Regards,

**Comment by jfoenixadmin** [comment]
Hello,
I pushed the fix github, it will be pushed to maven central in the next release. meanwhile you can get it by downloading jfoenix jar from the website.

you can now change the icon color using:
```
.jfx-date-picker > .arrow-button > .jfx-svg-glyph{
    -fx-background-color: WHITE;
}
```
instead of changing the default color property

Regards,
